packages/backend-core/src/api/resources/Organization.ts

```diff
@@ -0,0 +1,23 @@
+import camelcaseKeys from 'camelcase-keys';
+
+import filterKeys from '../utils/Filter';
+import type { OrganizationJSON } from './JSON';
+import type { OrganizationProps } from './Props';
+
+interface OrganizationPayload extends OrganizationProps {}
+
+export class Organization {
+  static attributes = ['id', 'name', 'createdAt', 'updatedAt'];
```

Comment:
❓ Do we get any nested organization attributes from the Backend API call or is it just there four?

Reply:
It's just these four organization attributes for now.